.python-version

```diff
@@ -0,0 +1 @@
+3.9.0
```

Comment:
so far tests were using 3.9.12 and lately 3.9.13. Is there any reason why you chose 3.9.0? Otherwise either we stick to 3.9.13 or 3.9 to use latest

```suggestion
3.9
```

Reply:
no reason at all, I did this because on my local machine I have a global .python-version set to 3.9.0 and I've just used that since it was working.

Looks like .python-version requires a full version tho:

```
~/s/a/alfresco-ansible-deployment ❯❯❯ pyenv versions
pyenv: version `3.9' is not installed (set by /Users/Giovanni.Toraldo/src/alfresco/alfresco-ansible-deployment/.python-version)
  system
  3.10.0
  3.9.0
```